FirebaseRemoteConfig/Sources/RCNConfigSettings.m

```diff
@@ -310,9 +272,6 @@ - (void)updateFetchTimeWithSuccessFetch:(BOOL)isSuccessfulFetch {
 }
 
 - (void)updateMetadataTable {
-  [_DBManager deleteRecordWithBundleIdentifier:_bundleIdentifier
-                                     namespace:_FIRNamespace
-                                  isInternalDB:NO];
```

Comment:
I think we still need this? so can't fully remove `deleteRecordWithBundleIdentifier` yet?

Reply:
I see. Good catch! Thanks.

The non-internal metadata path should now be restored.